### PR TITLE
add; check and uncheck todo list item

### PR DESCRIPTION
**Pull request by pedrosilva410**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -9,12 +9,24 @@ const todoList = document.getElementById('todoList');
...
 	if (userInput.value === '') {
-		alert('You really got nothing to do?')
+		alert('You really got nothing to do?');
 	}
 	else {
 		const li = document.createElement('li');
 		li.innerHTML = userInput.value;
-		todoList.appendChild(li)
+		todoList.appendChild(li);
+		let span = document.createElement('span');
```
Comment (201flaviosilva) [inline src/index.js]: This could be a `const`

Comment (Gabriel-Sous-a) [inline src/index.js]:
```suggestion
		const span = document.createElement('span');
```

File: src/index.js [hunk 2]
```diff
@@ -9,12 +9,24 @@ const todoList = document.getElementById('todoList');
...
 	}
 }
+
+todoList.addEventListener('click', function (e) {
+	if (e.target.tagName === 'LI') {
+		e.target.classList.toggle('checked');
+	}
+	else if (e.target.tagName === 'SPAN') {
+		e.target.parentElement.remove();
+	}
```
Comment (Gabriel-Sous-a) [inline src/index.js]:
```suggestion
	if (e.target.tagName === 'LI') {
		e.target.classList.toggle('checked');
	} else if (e.target.tagName === 'SPAN') {
		e.target.parentElement.remove();
	}
```

File: src/index.js [hunk 3]
```diff
@@ -9,12 +9,24 @@ const todoList = document.getElementById('todoList');
...
+		todoList.appendChild(li);
+		let span = document.createElement('span');
+		span.innerHTML = '\u00d7';
+		li.appendChild(span);
 		userInput.value = ''
 	}
 }
+
+todoList.addEventListener('click', function (e) {
+	if (e.target.tagName === 'LI') {
```
Comment (201flaviosilva) [inline src/index.js:29]: Here you could use a `switch`, `case`: https://www.w3schools.com/js/js_switch.asp

**Review by Gabriel-Sous-a — CHANGES_REQUESTED** [review]

File: src/index.js [hunk 4]
```diff
@@ -9,12 +9,24 @@ const todoList = document.getElementById('todoList');
 
 function addItem() {
```
Comment (201flaviosilva) [inline src/index.js]: This should be in the form submit, example:

```js
document.getElementById("MyFormID").addEventListener("submit", (event) => {
  event.preventDefault();
  const inputContent = document. getElementById("MyInputID").value;
});
```

File: src/index.js (hunk 3 above)
Comment (201flaviosilva) [reply src/index.js:29]: You should not check by tag name, you should add an event in each element

File: src/index.js (hunk 1 above)
Comment (pedrosilva410) [reply src/index.js]: okok

File: src/index.js (hunk 2 above)
Comment (pedrosilva410) [reply src/index.js]: tive 5 minutos a perceber a diferenca lol

File: src/index.js (hunk 1 above)
Comment (pedrosilva410) [reply src/index.js]: done

File: src/index.js (hunk 3 above)
Comment (pedrosilva410) [reply src/index.js:29]: yeah im doing that tomorrow

File: index.html [hunk 5]
```diff
@@ -14,10 +14,10 @@
 			<h1>Todo List</h1>
 			<time id="todayDate"></time>
 		</div>
-		<div class="input">
+		<form id="taskForm" onsubmit="addTask(event)">
```
Comment (201flaviosilva) [inline index.html:17]: This is the old html/js.

This it's the best way to do it, you should search a little about eventListerns: https://www.w3schools.com/js/js_htmldom_eventlistener.asp

```suggestion
		<form id="taskForm">
```

Comment (201flaviosilva) [reply index.html:17]: `submit` it's a event

File: src/styles.css [hunk 6]
```diff
@@ -115,6 +128,22 @@ ul li.checked::before {
   background-image: url(./images/checked.png);
 }
 
+ul li span {
+  position: absolute;
+  right: 8px;
+  top: 6px;
+  width: 30px;
+  font-size: 20px;
+  color: #555;
```
Comment (201flaviosilva) [inline src/styles.css:137]: You should use CSS variables, but it's total up to you: https://www.w3schools.com/css/css3_variables.asp

File: index.html (hunk 5 above)
Comment (pedrosilva410) [reply index.html:17]: okok

**Pull request merged**